pages/linux/csplit.md

```diff
@@ -0,0 +1,24 @@
+# csplit
+
+> Split a file into pieces.
+> This generates files named "xx00", "xx01", and so on.
+
+- Split a file at lines 5 and 23:
+
+`csplit {{file}} {{5}} {{23}}`
+
+- Split a file every 5 lines (this will fail if the total number of lines is not divisible by 5):
+
+`csplit {{file}} {{5}} {*}`
+
+- Split a file every 5 lines, ignoring exact-division error:
+
+`csplit -k {{file}} {{5}} {*}`
```

Comment:
yes, it is `--keep-files`. Should I write that in full in the code line, the description line, or both?

Reply:
Hm, that's... unexpected. Let me see if I understand the semantics of that option name: without the `-k` the partial files are created, but then deleted once the final one generates an error? Otherwise, how do you explain the choice of name? It's not clear to me.